simulators/main/shiftplan.py

```diff
@@ -1,20 +1,21 @@
+import logging
+import interface
+
 from datetime import datetime, timedelta
 from task import PeriodicTask
-
 from cumulocityAPI import CumulocityAPI
 from oeeAPI import OeeAPI 
-import logging
-import interface
+
 
 log = logging.getLogger("task")
 
 cumulocityAPI = CumulocityAPI()
 oeeAPI = OeeAPI()
 
-one_day = 86400 
+once_per_day = 86400
 
 class Shiftplan:
-    polling_interval = one_day
+    polling_interval = once_per_day
```

Comment:
I think we should revert that change b/c an "interval" defines "a period of time between events or states"; if we use "once_per_day" the variable should be named something like "polling rate"...